spec/prog/vm/nexus_spec.rb

```diff
@@ -352,6 +352,7 @@ def requested_disk_size(st)
         allocation_state_filter: ["accepting"],
         distinct_storage_devices: false,
         host_filter: [],
+        host_exclusion_filter: [],
```

Comment:
this basically checks that by default `host_exclusion_filter` is empty, and only set if `exclude_host_ids` is set on the stack, see `it "can exclude hosts"`. this can probably be improved, though.